frappe_graphql/graphql.py

```diff
@@ -13,6 +14,7 @@ def execute(query=None, variables=None, operation_name=None):
         variable_values=variables,
         operation_name=operation_name,
         field_resolver=default_field_resolver,
+        middleware=[disable_introspection_queries],
```

Comment:
maybe we can allow users to extend middle-wares through hooks like we do others ?

Reply:
@assemmarwan ?

Reply:
Good idea 👍🏻 , yet I think we can have that part of another PR